docs/release-notes/RELEASE-NOTES-1.4.0.md

```diff
@@ -29,7 +29,7 @@ You can run a node by either:
 [docker]: https://github.com/aeternity/aeternity/blob/v1.4.0/docs/docker.md
 [build]: https://github.com/aeternity/aeternity/blob/v1.4.0/docs/build.md
 
-The user configuration is documented in the [wiki](https://github.com/aeternity/aeternity/wiki/User-provided-configuration).
+The user configuration is documented in the [the dedicated separate document](../../docs/configuration.md).
```

Comment:
@zp-sd Re configuration.md linked twice:
It makes sense to remove this line, as below in this document the sentence "Configuring your node" refers to configuration.md
Though when removing this line please re-place the sentence below ` For specifying configuration using the Docker image, please refer to [its documentation][docker]. `.